configs/ath79-aruba-ap-105.config

```diff
@@ -0,0 +1,12 @@
+CONFIG_TARGET_ath79=y
+CONFIG_TARGET_ath79_generic=y
+CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_aruba_ap-105=y
+CONFIG_TARGET_PROFILE="DEVICE_aruba_ap-105"
+
+#
+# Aruba use initramfs to install
+#
+CONFIG_TARGET_ROOTFS_INITRAMFS=y
+
+CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_aruba_ap-105=""
+
```

Comment:
I'm not sure what this file is for. Is it used in the default build process somehow?